crates/ironrdp-dvc/src/lib.rs

```diff
@@ -1,4 +1,3 @@
-#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Intentional?

Reply:
Yes, although I figured out I can do this instead: 3d9462d9abadedadf81e4fcf91b6918ecd21a759